code/modules/food_and_drinks/recipes/recipes_grill.dm

```diff
@@ -124,39 +124,38 @@
 	)
 	result = /obj/item/food/snacks/wingfangchu
 
-/datum/recipe/grill/human/kabob
+/datum/recipe/grill/human/kebab
```

Comment:
I'm pretty sure this needs a proc override to rename the resulting kebab to include the name(s) of who it was made from.

Reply:
It's a subtype of an arbitrary human food found in sandwiches.dm, which also handles human burgers.